cf-serverd/server.c

```diff
@@ -241,7 +241,7 @@ static void SpawnConnection(EvalContext *ctx, const char *ipaddr, ConnectionInfo
         Log(LOG_LEVEL_ERR,
             "Unable to spawn worker thread. (pthread_create: %s)",
             GetErrorStr());
-        goto cleanup;
+        /* Execute cleanup: which is the next instruction. */
```

Comment:
Why change this? To the compiler it is essentially equivalent, and the old way would prevent someone from making a mistake if they add code under this block.